source/configuration_manual/config_file/config_file_syntax.rst

```diff
@@ -264,4 +264,10 @@ For example this is a common problem:
 
 This is because the second mail_plugins change that added `quota` globally didn't update anything inside the existing `protocol { .. }` or other filters.
 
-Some variables exist in the plugin section only, such as `sieve_extensions`. Those variables cannot be referred to, that is `$sieve_extensions` won't work.
\ No newline at end of file
+Some variables exist in the plugin section only, such as `sieve_extensions`. Those variables cannot be referred to, that is `$sieve_extensions` won't work.
+
+.. versionadded:: v2.3.14
+
+It is possible use ``$ENV:name`` to expand values from environment.
```

Comment:
I think would be clearer to have "Environment variable expansion" title here in the middle.